osquery/tables/system/windows/certificates.cpp

```diff
@@ -203,7 +203,17 @@ void enumerateCertStore(const HCERTSTORE& certStore,
                     certContext->pCertInfo->SerialNumber.pbData +
                         certContext->pCertInfo->SerialNumber.cbData),
         back_inserter(serial));
-    r["serial"] = serial;
+//serial number was in wrong endian, so reversed the string and then swapped 
```

Comment:
You'll wanna space these comments, but I believe you might want to just `clang-format` your commit and you should be good. You can do so with:
`clang-format -i osquery/tables/system/windows/certificates.cpp`